assets/js/index.js

```diff
@@ -377,6 +377,7 @@ function runDynamicCalculationsOnAdd(evt) {
     insertTimeBreaks(mod);
     calculateTime();
     calculateSummary();
+    initiateTimeEdit();
```

Comment:
this has to be removed right? as far as I tested the fix for #81 works and does not affect the time duration?

Reply:
ah sorry this is solved, as it was pointing to main and not development.